target/linux/realtek/base-files/etc/board.d/02_network

```diff
@@ -8,6 +8,7 @@ ucidef_set_poe() {
 		json_add_string "budget" "$1"
 		json_select_array ports
 			for port in $2; do
+				[ -n "$3" ] && [ -z "${3##*$port*}" ] && continue # skip ports passed via $3
```

Comment:
Call me a C bastard, but my eyes miss the `continue` statement after the logical predicate. I find the following form more readable:
```
			if [ -n "$3" ] && [ -z "${3##*$port*}" ]; then
				continue
			fi
```

Reply:
Fine with me.